docroot/themes/humsci/humsci_traditional/humsci_traditional.libraries.yml

```diff
@@ -6,6 +6,7 @@ base:
     js/index.js: {}
   dependencies:
     - core/modernizr
+    - classy/messages
```

Comment:
classy is deprecated and removed in D10: https://www.drupal.org/node/3313997

Whatever this helped with, please review an alternative.

Reply:
Fixed